src/main/resources/data/computerscience.json

```diff
@@ -0,0 +1,74 @@
+[
+  {
+    "name": "Computer Science",
+    "modules": [
+      { "name": "CS1010", "types": ["PR_F"] },
+      { "name": "CS1231", "types": ["PR_F"] },
+      { "name": "CS2030", "types": ["PR_F"] },
+      { "name": "CS2040", "types": ["PR_F"] },
+      { "name": "CS2100", "types": ["PR_F"] },
+      { "name": "CS2103T", "types": ["PR_F"] },
+      { "name": "CS2105", "types": ["PR_F"] },
+      { "name": "CS2106", "types": ["PR_F"] },
+
+      { "name": "CS3230", "types": ["PR_IER"] },
+      { "name": "CP3880", "types": ["PR_IER"] },
+      { "name": "CP3200", "types": ["PR_IER"] },
+      { "name": "CP3202", "types": ["PR_IER"] },
+      { "name": "IS4010", "types": ["PR_IER"] },
+      { "name": "CP3200", "types": ["PR_IER"] },
+      { "name": "CP3107", "types": ["PR_IER"] },
+
+      { "name": "IS1103/X" , "types": ["PR_IT"] },
+      { "name": "CS2101", "types": ["PR_IT"] },
+      { "name": "ES2660", "types": ["PR_IT"] },
+
+      { "name": "MA1521", "types": ["PR_MS_M"] },
+      { "name": "MA1101R", "types": ["PR_MS_M"] },
+      { "name": "ST2334", "types": ["PR_MS_M"] },
+      { "name": "PC1221/X", "types": ["PR_MS_S"] },
+
+      { "name": "CS3203", "types": ["PR_BD"] },
+      { "name": ["CS3216", "CS3217"], "types": ["PR_BD"] },
+      { "name": ["CS3281", "CS3282"], "types": ["PR_BD"] },
+
+      { "name": "GEH", "types": ["ULR"] },
+      { "name": "GER", "types": ["ULR"] },
+      { "name": "GET", "types": ["ULR"] },
+      { "name": "GEQ", "types": ["ULR"] },
+      { "name": "GES", "types": ["ULR"] },
```

Comment:
Every major takes `GE` modules as ULR. These 5 codes are not valid module codes by themselves. Let's keep `name` to always be valid module code that you can match with `ModuleInfo.getFromModuleCode`. We can code GE module's logic separately.

Reply:
Oops sorry forgot to add in the numbers